frontend/components/AddBookForm/AddBookForm.js

```diff
@@ -29,112 +29,109 @@ const AddBookForm = ({ onClick }) => {
+          type: 'success',
+          message: 'The Book ' + title + ' was succesfully added to the Library'
         });
         //I want the form to close after submit, so make sure this happens
         onClick();
       })
-      .catch((error) => {
+      .catch(error => {
         setToast({
-          type: "error",
+          type: 'error',
           message:
-            "The Book " +
+            'The Book ' +
             title +
-            "FAILED to added to the Library. ERROR: " +
-            error,
+            'FAILED to added to the Library. ERROR: ' +
+            error
         });
       });
   };
 
   return (
-    <StyledForm onSubmit={handleSubmit(submitForm)} data-testid={"form-1"}>
+    <StyledForm onSubmit={handleSubmit(submitForm)} data-testid={'form-1'}>
       <Flex
-        direction={"column"}
-        wrap={"nowrap"}
-        justifyContent={"center"}
-        borderRadius={"15px"}
+        direction={'column'}
+        wrap={'nowrap'}
+        justifyContent={'center'}
+        borderRadius={'15px'}
       >
         <SingleLineInput
-          labelText={"Title"}
+          labelText={'Title'}
           register={register}
           errors={errors}
-          width={"100%"}
+          width={'100%'}
         />
         <Flex
-          direction={"row"}
-          justifyContent={"flex-start"}
-          wrap={"nowrap"}
-          gap={"10px"}
+          direction={'row'}
+          justifyContent={'flex-start'}
+          wrap={'nowrap'}
+          gap={'10px'}
         >
           <SingleLineInput
-            labelText={"First Name"}
+            labelText={'First Name'}
             register={register}
             errors={errors}
           />
           <SingleLineInput
-            labelText={"Last Name"}
+            labelText={'Last Name'}
             register={register}
             errors={errors}
           />
         </Flex>
         <Flex
-          height={"200px"}
-          direction={"column"}
-          justifyContent={"flex-end"}
-          alignContent={"flex-end"}
+          height={'200px'}
+          direction={'column'}
+          justifyContent={'flex-end'}
+          alignContent={'flex-end'}
         >
           <MultiLineInput
-            labelText={"Description"}
+            labelText={'Description'}
             rows={5}
             register={register}
             errors={errors}
           />
         </Flex>
         <Flex
-          justifyContent={"flex-end"}
-          alignContent={"flex-end"}
-          height={"50px"}
-          padding={"30px 0px 0px 0px"}
-          gap={"10px"}
+          justifyContent={'flex-end'}
+          alignContent={'flex-end'}
+          height={'50px'}
+          padding={'30px 0px 0px 0px'}
+          gap={'10px'}
         >
           <Button
             fColor={Colors.Bright[1]}
             borderColor={Colors.Bright[1]}
-            label={"Cancel"}
+            label={'Cancel'}
             onClick={onClick}
```

Comment:
Take a look at your variable names, they should be readable in the sense that if I read the varariable name, I know exactly what it's doing.

For this one, a good alternative would be:
`handleSubmitForm` or something similar (use your appropriate context)

Reply:
ok i think i have made the requested modification. I added a text property to the Button class, that way i can name the label something different and more applicable to the situations.